transport/TransportImpl.cpp

```diff
@@ -2434,13 +2423,66 @@ void TransportImpl::setTag(const char* tag)
     utils::strncpy(_tag, tag, sizeof(_tag));
 }
 
+void TransportImpl::onIceCandidateAccepted(ice::IceSession* session,
+    ice::IceEndpoint* endpoint,
+    const ice::IceCandidate& remoteCandidate)
+{
+    logger::info("candidate accepted: %s %s, %s - %s",
```

Comment:
Maybe it is not consistent through code, but I think there was some reasoning that PII info like IP should only be logged at debug level. So it is generally not available in the cloud logs.